examples/bitcoind-rpc-client/src/bitcoind_client.rs

```diff
@@ -0,0 +1,118 @@
+use std::{sync::{Arc}};
```

Comment:
```suggestion
use std::{sync::{Arc}, io::{Result, Error, ErrorKind}};
```
Not sure if it's best practice. But it could avoid adding the namespace every time we call `Result` or `Err`.
So `std::io::Result` can be just `Result`. Thoughts?

Reply:
Agreed, much better for readability

Reply:
Don't want to bother you too much as I'm not involved here, just curious about the project and randomly noticed this. I think importing `std::io::Result` is much worse for readability because it changes type - `Result` normally has two type parameters. Also `std::io::Error` could be confused with `std::error::Error`. I personally always import the `std::io` module and then write `io::Result`, `io::Error` and `io::ErrorKind`. It's still reasonably concise and clear that those types are IO-related.

I strongly recommend you do the same. I vaguely remember even Rust developers admitting naming `Result` in `io` might have been a mistake. If you don't like double colon `use std::io::Result as IoResult` should be also fine. (I personally prefer double colon.)

Reply:
Makes sense. And going by the project coding style, it looks like folks have been using `io::Error` , `fmt::Error`.
So importing just `std::io` might be the sweet spot.
@ConorOkus I agree with @Kixunil suggestion. You may change the line to 
```
use std::{sync::{Arc}, io};
``` 
And use just `io::Error` ? Let me know what you feel.

Reply:
Great suggestions, i'll update